sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -1457,6 +1459,19 @@ class HistGradientBoostingClassifier(ClassifierMixin, BaseHistGradientBoosting):
 
         .. versionadded:: 0.24
 
+    See Also
+    --------
+    GradientBoostingClassifier : Gradient Boosting for classification.
```

Comment:
If think it would be nice here to mention that this is the "exact" method but that it does not scale as well as HGDT on large samples dataset.